src/Bicep.Core.IntegrationTests/MsGraphTypesViaRegistryTests.cs

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.IO.Abstractions.TestingHelpers;
+using Azure;
+using Bicep.Core.Configuration;
+using Bicep.Core.Diagnostics;
+using Bicep.Core.Registry;
+using Bicep.Core.Semantics.Namespaces;
+using Bicep.Core.TypeSystem.Providers;
+using Bicep.Core.UnitTests;
+using Bicep.Core.UnitTests.Assertions;
+using Bicep.Core.UnitTests.Mock;
+using Bicep.Core.UnitTests.Registry;
+using Bicep.Core.UnitTests.Utils;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using RegistryUtils = Bicep.Core.UnitTests.Utils.ContainerRegistryClientFactoryExtensions;
+
+namespace Bicep.Core.IntegrationTests
+{
+
+    [TestClass]
+    public class MsGraphTypesViaRegistryTests : TestBase
+    {
+        private const string versionV10 = "1.2.3";
```

Comment:
Question - how are we going to distinguish between `v1.0` and `v10`? I know it's not likely that there will be a `v10`, but just curious if `v10` would cause confusion.

Reply:
Nvm just realized we are just using this in the code. Customers can choose whatever they want for the symbolic name in the bicepconfig.json file.

Reply:
Yeah this is just variable naming for test. Registry artifacts are not published yet but ideally public naming should always be v1.0. `v1` is referenced in a few places in the test and will update these once we have the v1.0 repo path reserved and approved in mcr.